content/documentation/pages/5-feature-guides/3-batch/4-monitoring.md

```diff
@@ -124,6 +132,18 @@ And in [Grafana dashboard for Tasks](http://localhost:3000/d/scdf-tasks/tasks?re
 
 ![SCDF Task Grafana Prometheus Dashboard](images/SCDF-task-metrics-grafana-prometheus-dashboard.png)
 
+<!--Wavefront-->
+
+### Wavefront
+
+To install Data Flow with Wavefront support, follow the [Monitoring with Wavefront](%currentPath%/installation/local/docker-customize/#wavefront) Docker Compose instructions. This will bring up Spring Cloud Data Flow, Skipper, Apache Kafka, and point it to the Wavefront Data Flow Integration Tile.
```

Comment:
```suggestion
To install Data Flow with Wavefront support, follow the [Monitoring with Wavefront](%currentPath%/installation/local/docker-customize/#wavefront) Docker Compose instructions. This will bring up Spring Cloud Data Flow, Skipper, Apache Kafka, and it will also point to the Wavefront's Data Flow Integration Tile automatically.
```